src/Linkify.jsx

```diff
@@ -13,7 +13,9 @@ class Linkify extends React.Component {
     component: React.PropTypes.any,
     properties: React.PropTypes.object,
     urlRegex: React.PropTypes.object,
-    emailRegex: React.PropTypes.object
+    emailRegex: React.PropTypes.object,
+    preCompileURL: React.PropTypes.func,
+    preCompileText: React.PropTypes.func
```

Comment:
Why the word "compile"? 
To me, that word means you're preparing something *before* you can use it. Also, the word "pre" is nice but it's jarring to me that there's no "post".